testsuite/forge/src/backend/k8s/node.rs

```diff
@@ -77,12 +77,16 @@ impl Node for K8sNode {
     }
 
     fn health_check(&mut self) -> Result<(), HealthCheckError> {
-        let results = self
+        let health_error = Err(HealthCheckError::RpcFailure(format_err!("")));
+        let results = match self
             .runtime
             .block_on(self.json_rpc_client().batch(Vec::new()))
-            .unwrap();
+        {
+            Ok(x) => x,
+            Err(_) => return health_error,
```

Comment:
Might be best to actually take the error and return it instead of returning a dummy one?